ts/Core/Axis/AxisDefaults.ts

```diff
@@ -1538,15 +1538,29 @@ namespace AxisDefaults {
 
         /**
          * Additional range on the right side of the xAxis. Works similar to
-         * `xAxis.maxPadding`, but value is set in milliseconds. This property
-         * should be set to the same value for both the main `xAxis` and the
+         * `xAxis.maxPadding`, but value is set in milliseconds, percentage or pixels.
+         *
+         * If it's a number, it is interpreted as milliseconds.
+         *
+         * If it's a percentage string,
+         * it is interpreted as percentages of axis length.
+         * An overscroll of 50% will make a 100px axis 50px longer.
+         *
+         * If it's a pixel string, it is interpreted as fixed pixel value,
+         * but limited to 90% of axis width.
+         *
+         * This property should be set to the same value for both the main `xAxis` and the
          * navigator's `xAxis` to maintain consistent behavior during
          * interactions such as dragging or resizing the navigator.
```

Comment:
Did we consider having the navigator x-axis inherit the `overscroll` of the parent by default? This seems like ouble work for a very common situation. @pawelfus

Reply:
Good point. Yes, we did and resigned for some (unknown) reason. Found the related commit: https://github.com/highcharts/highcharts/commit/59108d0b219a7eae081f8f061142ec5ce9a886e6 - no test, so we can only guess and test it manually before implementing

Reply:
@jakubjanuchta while resolving other issues, could you test it? It should be as simple as inheriting this option in Navigator.ts in `init` (where new Axis is being created)